packages/contracts/contracts/core/DAO.sol

```diff
@@ -196,10 +201,14 @@ contract DAO is IDAO, Initializable, UUPSUpgradeable, ACL, ERC1271, AdaptiveERC1
         _handleCallback(msg.sig, msg.data); // WARN: does a low-level return, any code below would be unreacheable
     }
 
+    /// @notice Method that emmits the MetadataSet event when a new metadata is set
+    /// @param _metadata Hash of the IPFS file object 
     function _setMetadata(bytes calldata _metadata) internal {
         emit MetadataSet(_metadata);
     }
 
+    /// @notice set trusted forwarder on the DAO and emits the event
```

Comment:
Sometimes, the start of the sentence is capitalized, sometimes not. 
Also, sometimes, the event name is explicitly mentioned.
Let's do this uniformly.